vcluster/deploy/control-plane/binary/readme.mdx

```diff
@@ -0,0 +1,278 @@
+---
+title: Standalone vCluster
+sidebar_label: Standalone
+sidebar_position: 2
+sidebar_class_name: pro
+---
+
+
+## vCluster Standalone
+
+vCluster Standalone is a different architecture model for vCluster. By default, the control plane and worker nodes are deployed on shared nodes from a host cluster, but with vCluster Standalone, there is no longer a host cluster. vCluster is deployed directly onto nodes like any typical Kubernetes distribution. vCluster Standalone can run on any type of node, whether that is a bare-metal node or VM. It doesn't matter where the node is hosted.
+
+It provides the strictest isolation for workloads as there is no shared host cluster for the control plane or worker nodes. When enabling vCluster Standalone, the control plane is now no longer on a shared host cluster, but on its own independent node. To add worker nodes, the private nodes feature is required.
+
+<br />
+
+<center>
+  <img
+    src="/docs/media/private-nodes/architecture.png"
+    width='500'
+    alt="vCluster Standalone Architecture"
+  />
+</center>
+
+You can connect and manage vCluster Standalone via vCluster Platform.
```

Comment:
We should remove this, will cause problems. We should actually say to not add it to platform at this point.